README.md

```diff
@@ -9,48 +9,48 @@ Go implementation of Waves Node, libraries and tools for Waves blockchain.
 
 ## Waves Node
 
-It is possible to run Waves Node on Linux, macOS or Windows. Please, download the appropriate binary file from [Releases page](https://github.com/wavesplatform/gowaves/releases).  
+It is possible to run Waves Node on Linux, macOS or Windows. Please, download an appropriate binary file from [Releases page](https://github.com/wavesplatform/gowaves/releases).
 
 You can either synchronize a node over network or import a downloaded blockchain file.
 
 ### How to import blockchain from file
 
-Blockchain files are available at [MainNet](http://blockchain.wavesnodes.com) and [TestNet](http://blockchain-testnet.wavesnodes.com) download pages.
+Blockchain files are available on [MainNet](http://blockchain.wavesnodes.com), [TestNet](http://blockchain-testnet.wavesnodes.com) [StageNet](http://blockchain-stagenet.wavesnodes.com/) download pages.
 
 Import could be done as follows:
 
- 1. Download a blockchain file
- 1. Download the `importer` utility from [Releases](https://github.com/wavesplatform/gowaves/releases)
- 1. Run the command passing the paths to blockchain file and node's state directory as parameters. 
- The third parameter is the number of blocks to import, it should be less than desired height by one.  
- 
+1. Download a blockchain file
+1. Download the `importer` utility from [Releases](https://github.com/wavesplatform/gowaves/releases)
+1. Run the command, put the path to the blockchain file and node's state directory as parameters.
+   The third parameter is the number of blocks to import, it should be less than a desired height by one.
+
 ```bash
 ./importer -blockchain-path [path to blockchain file] -data-path [path to node state directory] -blocks-number [height - 1]
 ```
 
-Import could take up a few hours, afterward run the node as described in next section. 
+Import may take a few hours, after which you can run the node as described in next section.
 
-Note that the Go node has its own state storage structure that is incompatible with Scala Node.
+Please mote that the Go Node has its own state storage structure that is incompatible with Scala Node.
 
-### How to run node
+### How to run the node
 
 Run the node as follows:
 
- 1. Download the suitable node's binary from [Releases](https://github.com/wavesplatform/gowaves/releases) 
- 1. Run the command, if required, pass the path to the node's state directory.
- 
+1. Download a suitable node's binary file from [Releases](https://github.com/wavesplatform/gowaves/releases)
+1. Run the command, if required, put the path to the node's state directory.
+
 ```bash
 ./node -state-path [path to node state directory]
 ```
 
-By default the node starts as MainNet node. To start TestNet node pass the `testnet` as blockchain type and comma separated list of TestNet peer's addresses:
+By default, the node is run as a MainNet node. To run a TestNet node put `testnet`, as a blockchain type, and a list of comma separated TestNet peer's addresses:
```

Comment:
Need to mention that peers addresses is optional or remove part about peers addresses